Resources/Prototypes/_Goobstation/Damage/modifier_sets.yml

```
@@ -29,4 +29,5 @@
     Shock: 1
     Cold: 0.7  # Fur = Keep warm
     Bloodloss: 0.9
-    Poison: 0.5 # Should be more resistant to toxin
\ No newline at end of file
+    Poison: 0.5 # Should be more resistant to toxin
+    Radiation : 0.7 # Engie yowies need some help against anti-rads
```

yamllint

[warning] too many spaces before colon

(colons)